life_expectancy/tests/test_load_save_data.py

```diff
@@ -0,0 +1,18 @@
+"""Tests for the load and save module"""
+from unittest.mock import patch
+import unittest.mock as mock # pylint: disable=consider-using-from-import
```

Comment:
```suggestion
```

I didn't see this being used...

Reply:
is being used below on 'with'

Reply:
So que está a usar, na verdade o `mock.patch`. Como já estás as importar o `patch`, poderias simplesmente usar `with patch`, correto?

Reply:
eu fiz isso mas fica a dar-me erro 😭

Reply:
Qual é o erro que dá?

Reply:
deixou de dar 🫥mas a serio que estava a dar 🤔

Reply:
ah ham 😏